services/movie.py

```diff
@@ -0,0 +1,37 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids:
+        queryset = queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(actors__id__in=actors_ids)
+    if actors_ids and genres_ids:
+        queryset = queryset.filter(
+            genres__id__in=genres_ids).filter(
+            actors__id__in=actors_ids)

```

Comment:
I don't think it's necessary to have an additional check for both actors_ids and genres_ids